### PR TITLE
Uplift final

### PR DESCRIPTION
**Pull request by travis-minke-sap**
Final uplift to eventing-contrib module dependency versions in preparation for bringing over eventing-contrib/kafka content.

### CODE REVIEW

File: pkg/channel/distributed/common/kafka/admin/util/util.go [hunk 1]
```diff
@@ -1,6 +1,8 @@
 package util
 
 import (
+	"context"
+
```
Comment (mattmoor) [inline pkg/channel/distributed/common/kafka/admin/util/util.go:5]: Remove trailing whitespace:
```suggestion

```

File: pkg/channel/distributed/controller/kafkasecret/kafkachannel.go [hunk 2]
```diff
@@ -1,10 +1,13 @@
 package kafkasecret
 
 import (
+	"context"
 	"fmt"
+
```
Comment (mattmoor) [inline pkg/channel/distributed/controller/kafkasecret/kafkachannel.go:6]: Remove trailing whitespace:
```suggestion

```

File: pkg/channel/distributed/controller/kafkasecretinformer/kafkasecretinformer.go [hunk 3]
```diff
@@ -3,16 +3,17 @@ package kafkasecretinformer
 import (
 	"context"
 	"fmt"
+
```
Comment (mattmoor) [inline pkg/channel/distributed/controller/kafkasecretinformer/kafkasecretinformer.go:6]: Remove trailing whitespace:
```suggestion

```

File: pkg/channel/distributed/common/kafka/admin/util/util.go (hunk 1 above)
Comment (mattmoor) [inline pkg/channel/distributed/common/kafka/admin/util/util.go:5]: Format Go code:
```suggestion

```

File: pkg/channel/distributed/controller/kafkasecret/kafkachannel.go (hunk 2 above)
Comment (mattmoor) [inline pkg/channel/distributed/controller/kafkasecret/kafkachannel.go:6]: Format Go code:
```suggestion

```

File: pkg/channel/distributed/controller/kafkasecretinformer/kafkasecretinformer.go (hunk 3 above)
Comment (mattmoor) [inline pkg/channel/distributed/controller/kafkasecretinformer/kafkasecretinformer.go:6]: Format Go code:
```suggestion

```

File: pkg/channel/distributed/controller/kafkasecretinformer/kafkasecretinformer_test.go [hunk 4]
```diff
@@ -2,11 +2,12 @@ package kafkasecretinformer
 
 import (
 	"context"
+
```
Comment (mattmoor) [inline pkg/channel/distributed/controller/kafkasecretinformer/kafkasecretinformer_test.go:5]: Format Go code:
```suggestion

	"testing"

```

File: pkg/channel/distributed/controller/kafkasecretinformer/kafkasecretinformer_test.go [hunk 5]
```diff
@@ -2,11 +2,12 @@ package kafkasecretinformer
...
 	"github.com/stretchr/testify/assert"
 	"k8s.io/client-go/kubernetes/fake"
-	"knative.dev/eventing/pkg/logging"
 	injectionclient "knative.dev/pkg/client/injection/kube/client"
 	"knative.dev/pkg/injection"
+	"knative.dev/pkg/logging"
 	logtesting "knative.dev/pkg/logging/testing"
 	_ "knative.dev/pkg/system/testing"
 	"testing"
```
Comment (mattmoor) [inline pkg/channel/distributed/controller/kafkasecretinformer/kafkasecretinformer_test.go:13]: Format Go code:
```suggestion

```

**Review by mattmoor — COMMENTED** [review]
Produced via:
  `gofmt -s -w $(find -path './vendor' -prune -o -path './third_party' -prune -o -name '*.pb.go' -prune -o -type f -name '*.go' -print)`
  `goimports -w $(find -name '*.go' | grep -v vendor | grep -v third_party | grep -v .pb.go | grep -v wire_gen.go)`

---
Unable to apply comments to the following files: 
 - `pkg/channel/distributed/common/kafka/admin/eventhubcache/cache.go`
 - `pkg/channel/distributed/controller/kafkachannel/reconciler_test.go`
 - `pkg/channel/distributed/controller/kafkasecret/reconciler_test.go`
 - `pkg/channel/distributed/controller/kafkasecretinjection/reconciler.go`

**Review by eric-sap — APPROVED** [review]
looks good

**Review by matzew — APPROVED** [review]
/lgtm
/approve

**Pull request merged**